components/org.wso2.carbon.identity.api.user.fido2/org.wso2.carbon.identity.api.user.fido2.v1/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Expose the API in https://github.com/wso2/identity-rest-dispatcher @https://github.com/wso2/identity-rest-dispatcher/blob/master/components/org.wso2.carbon.identity.api.dispatcher/src/main/webapp/WEB-INF/beans.xml